lib/versions.js

```diff
@@ -59,6 +78,12 @@ function compareVersions(v1, v2) {
     if (semver.lt(v1.tag, v2.tag)) {
         return 1;
     }
+    if (v1.published_at  > v2.published_at) {
```

Comment:
Nope.
but that should be fine.
if version itself (without commit id) is higher, it will take highest.
else, it will take the latest published